Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "feat/CIV-5365"
```

Comment:
Presumably this will need to change before merging.

Reply:
Yes, this will have to be changed back to master before merging.